main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1442,8 +1442,14 @@ object Resolver {
 
         case NamedAst.Expr.InvokeStaticMethod2(clazzName, methodName, exps, loc) =>
           val esVal = traverse(exps)(visitExp(_, env0))
-          mapN(esVal) {
-            es => ResolvedAst.Expr.InvokeStaticMethod2(Class.forName(clazzName.name), methodName, es, loc)
+          flatMapN(esVal) {
+            es => env0.get(clazzName.name) match {
+              case Some(List(Resolution.JavaClass(clazz))) =>
+                Validation.success(ResolvedAst.Expr.InvokeStaticMethod2(clazz, methodName, es, loc))
+              case _ =>
+                val m = ResolutionError.UndefinedJvmClass(clazzName.name, "", loc)
```

Comment:
What's the ""? Should there be a TODO here?

Reply:
Its a mistake that it even takes a string, but we will deal with that later.